src/ray/common/scheduling/cluster_resource_data.h

```diff
@@ -304,8 +304,11 @@ class NodeResources {
   // The idle duration of the node from resources reported by raylet.
   int64_t idle_resource_duration_ms = 0;
 
-  // Whether the node is being drained or not.
-  bool is_draining = false;
+  // 0 if the node is not being drained.
```

Comment:
Hmm 0 == not being drained sounds weird. Isn't it more like 0 == disabled (no timeout)? Is there a better way to express "not being drained"?

Reply:
Added is_draining bool as well.